src/ecalc/libraries/libecalc/common/libecalc/dto/result/results.py

```diff
@@ -96,6 +96,10 @@ class GenericConsumerResult(EquipmentResultBase):
 
 class PumpResult(EquipmentResultBase):
     componentType: Literal[ComponentType.PUMP]
+    inlet_liquid_rate_m3_per_d: TimeSeriesRate
```

Comment:
```suggestion
    inlet_liquid_rate_m3_per_day: TimeSeriesRate
```

please spell fully out. we have space and time and auto-complete ;) ! :)